src/components/profile.vue

```diff
@@ -127,30 +151,39 @@
         }
       },
       submit: function(evt) {
-        console.log('TEST', evt.target.form_name.value)
-        localforage.getItem('X_TOKEN')
-          .then(session => {
-            axios.post(`/api/v1/profile/edit/${this.sessionInfo._id.$oid}`, {
-              headers: { 'x-token': session },
-              params: {
-                full_name: evt.target.form_name.value,
-                email: evt.target.form_email.value,
-                password: evt.target.form_password.value,
-              }
-          })
-          .then(res => {
-            console.log('DATA RETURNING', res.data)
-            this.sessionInfo = res.data
+        this.errors =[];
+        if(evt.target.form_name.value !== evt.target.form_con_name.value) {
+          this.errors.push('Name does not match');
+        }
+        if(evt.target.form_email.value !== evt.target.form_con_email.value) {
+          this.errors.push('Email does not match');
+        }
+        if(evt.target.form_password.value !== evt.target.form_con_password.value) {
+          this.errors.push('Password does not match');
+        }
+        if (this.errors.length === 0) {
+          localforage.getItem('X_TOKEN')
+            .then(session => {
+              axios.post(`/api/v1/profile/edit/${this.sessionInfo._id.$oid}`, {
+                headers: { 'x-token': session },
+                params: {
+                  full_name: evt.target.form_name.value.length !== 0 ? evt.target.form_name.value : this.sessionInfo.full_name,
+                  email: evt.target.form_email.value.length !== 0 ? evt.target.form_email.value : this.sessionInfo.email ,
+                }
+            })
+            .then(res => {
+              console.log('DATA RETURNING', res.data)
```

Comment:
Can we remove this console.log() call? Is there any reason for it to still be in here.